index.html

```diff
@@ -85,6 +85,191 @@ <h1 class="article">I'm a software developer.</h1>
           </ul>
         </div>
         <div class="space"></div>
+        <div>
+          <img
+            class="drop-down"
+            src="Images/drop-down.png"
+            alt="drop-down button"
+          />
+        </div>
+      </section>
+    </main>
+    <section class="rest">
+      <h1 class="works">My Recent Works</h1>
```

Comment:
[REQUIRED] for accessibility you are expected to use the `h1` tag only once in a document. Kindly change this to another heading tag